content/java/concepts/calendar/terms/equals/equals.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: '.equals()'
+Description: 'Adds the equals() method term entry for the Java Calendar class.'
+Subjects:
+  - 'Learn Java'
+Tags:
+  - 'Calendar'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+---
+
+
+
+# `equals()` Method in Java Calendar
+
+The `equals()` method in the Java `Calendar` class is used to determine if two calendar objects are equal. It compares the time values of the two calendar instances to check if they represent the same point in time.
+
+## Syntax
+
+```java
+public boolean equals(Object obj) {
+    // Implementation details for the equals() method
+}
+```
+# `equals()` Method in Java Calendar
```

Comment:
````suggestion
```pseudo
result = myCalendar.equals(obj)
```
````
The syntax example should be an outline of the method in use, not the method signature.